src/_includes/components/navigation-secondary.njk

```diff
@@ -14,19 +17,22 @@
   }
 </style>
 
-<nav aria-label="secondary">
-  <ul>
-    {%- for item in primaryNav %}
-      {% if page.url | startsWith(item.url) %}
-        <li><a class="text-semibold" href="{{ item.url | url }}">{{ item.title }}</a></li>
-        {%- if item.children %}
-          {%- for subitem in item.children %}
-            <li>
-              <a href="{{ subitem.url | url }}" {% if page.url | startsWith(subitem.url) %} aria-current="page"{% endif %}>{{ subitem.title }}</a>
-            </li>
-          {%- endfor %}
+{# Not sure about this if layout != home as it feels a bit flimsy #}
+{% if layout != 'home' %}
```

Comment:
Yeah, it's a bit of an odd structure the navigation plugin gives us here, but hierarchically good. 

Could we perhaps check that `secondaryNav` isn't null and/or has a length, and then determine if its secondary nav (ie. `primaryNav[0].children`) has length/subitems to iterate over?

Reply:
I would prefer a solution like this but I can't get it working.

When logging out the primaryNav there is always children, I don't think it's taking into consideration the home/root of the site.

Reply:
Let's revisit this as  it's a bit of an iceberg.